package.json

```diff
@@ -51,6 +51,9 @@
     "require": "ts-node/register/transpile-only",
     "_": "test/**/*.js"
   },
+  "engines": {
+    "node": ">=16.13.0"
+  },
```

Comment:
```suggestion
  "engines": {
    "node": ">=16.13.0",
    "npm": "=>8.1.0"
  },
```

drop
```
"prepublish": "require-npm4-to-publish",
...

"require-npm4-to-publish": "^1.0.0",
```

Reply:
@fisehara I've noticed `require-npm4-to-publish` in a number of `balena-io`/`balena-io-modules` repos and I feel like it shouldn't be required/used any longer? Especially if we require more recent versions of node/npm in `package.json`'s `engines` section. We always use this `engines` section in conjunction with [`engine-strict=true`](https://docs.npmjs.com/cli/v9/using-npm/config#engine-strict) in `.npmrc` in Jellyfish repos to enforce engine versions.

Also, by dropping `npm` version from `engines` config, am I right in thinking that the version of npm used is whatever the version is for the defined node version? Meaning node v16 should come bundled with npm v8.x.x. Making `require-npm4-to-publish` completely unnecessary.